### ISSUE TITLE
Delete & Create Appointment

### ISSUE THREAD

**Issue by Bman2022** [phw198/OutlookGoogleCalendarSync#1502]
**Are you already using OGCS?**
v2.9.0.0

**Describe the question you have**

Good morning.  Just started using OGCS - great program!  Question, I noticed that a calendar event is deleted and added to my Google calendar when a scheduled sync process occurs.  It doesn't appear to happen when I force a sync with Sync Now.   The event was created on my Outlook application in the linked Google calendar so I think that may have something to do with it.  Really this is a google event created in Outlook so it would be google syncing with google calendar.   The event appears as I would expect in both Outlook and Google but I don't understand the delete + create routine OGCS is going through during sync.   If I delete the requirement to confirm deletions, I wouldn't notice anything was going on with the sync so maybe this is just part of the normal sync process? 

Thanks
Benjamin

**Comment by phw198** [comment]
No, that doesn't sound like intended behaviour - but I'm not sure what you mean by "the event was created on my Outlook application in the linked Google calendar". Do you mean you have your Google calendar surfaced in Outlook? If you have manually created an item in Outlook calendar and _also_ in Google _through_ Outlook, and then also sync from Outlook to Google with OGCS, then yes, I would expect the latter to be deleted or "reclaimed" during sync.

Clicking "Sync Now" is performing exactly the same sync as a scheduled one does, so there shouldn't be any difference in behaviour.

If you would like the issue investigated further though, you would need to replicate it on the latest alpha release and provide logs.
